deployment/ansible/roles.yml

```diff
@@ -1,14 +1,5 @@
-- src: azavea.pip
-  version: 1.0.0
+- src: azavea.nodejs
```

Comment:
Out of curiosity, how frequently will this nodejs version change major versions?

Reply:
Does this help? https://github.com/nodejs/Release

Reply:
Oh -- I meant the azavea.nodejs role but it may be that there's no difference. Will the way this is set up also jump between LTS and non-LTS versions?

Reply:
We feed the NodeJS version in `deployment/ansible/group_vars/all` and the `azavea.nodejs` role installs whatever we specify.

Versioning for the role itself is just stuff like bug fixes and updating where to look for packages. 

See:

https://github.com/azavea/ansible-nodejs/blob/develop/CHANGELOG.md